questions/qa-headers-charset.en.html

```diff
@@ -80,37 +80,63 @@ <h2>The i18n Checker</h2>
     <p>The <a class="print" href="https://validator.w3.org/i18n-checker/">Internationalization Checker tool</a>, developed by the W3C, checks web pages for various internationalisation issues. It also has an information section that summarises key internationalization-related information about a page, such as character encoding and language declarations, etc. That section tells you whether an encoding declaration is used in the HTTP header, and if so, what is the encoding.</p>
     <p>The i18n checker tool is particularly useful, since it also shows you  other encoding declarations used in the document, and raises a flag if there are differences.</p>
   </section>
-  
-  
-  
-  
-  
+
+
+
+
+
   <section id="webbased">
 <h2>Use a Web-based service</h2>
     <p>There are several services that show you all the HTTP headers and the (HTML) source of the document returned from the server
       after you enter the address of the document you are interested in:</p>
     <ul>
-      <li><a class="print" href="http://www.delorie.com/web/headers.html">Delorie HTTP Header Viewer</a></li>
-      <li><a class="print" href="http://www.rexswain.com/httpview.html">Rex Swain's HTTP Viewer</a></li>
+      <li><a class="print" href="https://www.rexswain.com/httpview.html">Rex Swain's HTTP Viewer</a></li>
+      <li><a class="print" href="https://tools.keycdn.com/curl">HTTP Header Checker by KeyCDN</a></li>
     </ul>
     <p>Note: W3C has no relationship to any of these services.</p>
     <p>In the HTTP headers, look for the <code class="kw" translate="no">Content-Type</code> header, and in particular for the <code class="kw" translate="no">charset</code> parameter, e.g.</p>
     <blockquote><code>Content-Type: text/html; charset=utf-8</code></blockquote>
     <p>Note: The <code class="kw" translate="no">charset</code> parameter may not be present. This is okay if your document itself indicates its character
       encoding.</p>
   </section>
-  
+
+  <section id="devtools">
+    <h2>Use a browser DevTools</h2>
+        <p>Modern desktop browsers usually contain a set of web developer tools, and they can be used to view HTTP headers of a web document and its related resources. Here's how to do it in major browsers.</p>
+
+    <section id="devtools-1">
+      <h3>Chrome, Edge, Firefox, and Opera</h3>
+        <ol>
+          <li>Open DevTools by pressing <kbd>Control+Shift+I</kbd> or <kbd>Command+Option+I</kbd> (Mac).</li>
+          <li>Click the <strong>Network</strong> tab. Right now the Network panel is empty.</li>
+          <li>Reload the page to log some network activity. Each row represents a resource. The top resource is usually the main HTML document.</li>
+          <li>Click a resource and open the <strong>Headers</strong> tab to view the HTTP headers.</li>
+        </ol>
```

Comment:
Since this page is actually about finding the character encoding information, i'd suggest to add another list item to tell the reader to look for the `content-type` value. Otherwise, some readers may get confused by `content-encoding` or `accept-encoding` headers.

Reply:
Done. Thanks.